### DISCUSSION TITLE
Didn't recieved Github Student Pack yet despite approval on 28 December 2024

### DISCUSSION THREAD

**Discussion by msafeerhassan** [community/community#148157] [GitHub Education]
### Hi everyone,

I haven't recieved any email or any update about my Github Student Developer Pack. I sumbitted it on 28 December 2024 and its status is being shown as approved but still I am uncertain about it as I haven't recieved any email or notification yet from Github Team. Please help!!

![best-logo-2013-3](https://github.com/user-attachments/assets/a28868a9-a380-49a6-a913-6bf40e299201)

**Comment by aayush-ojha** [comment]
Hi @msafeerhassan,
You should normally get an email from GitHub Education after your application has been approved. Please recheck the email that you used while submitting for GitHub Education(example@organization.com). 

If you still haven't got any update you can check your benefits at:
https://education.github.com/learner/learn

**Comment by davevad93** [comment]
Hi @msafeerhassan ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @msafeerhassan was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
